src/fuzzers/compare_to_spec.rs

```diff
@@ -116,17 +116,20 @@ fn fuzz() {
             t.run();
             if t.failed() {
                 let _ = std::fs::write("/tmp/fuzz-crash.wasm", &bytes);
-                panic!(
+                let output = String::from_utf8_lossy(&t.output);
+                eprintln!(
                     "Test failed, module available in /tmp/fuzz-crash.wasm.
 Module bytes were:
 {:?}
 Wast test was:
 {}
 Test output:
 {}",
-                    bytes,
-                    wast_test,
-                    String::from_utf8_lossy(&t.output),
+                    bytes, wast_test, output,
+                );
+                panic!(
+                    "{}",
+                    itertools::intersperse(output.lines(), "; ").collect::<String>()
```

Comment:
Any reason you didn’t use [`slice::join`](https://doc.rust-lang.org/stable/std/primitive.slice.html#method.join)?

Reply:
I had no idea it existed, after seeing Iterator::intersperse being still unstable 😅